holoviews/ipython/__init__.py

```diff
@@ -179,18 +179,21 @@ def __call__(self, *args, **params):
                                                                               fig='svg')
                         holoviews.archive.exporters = [svg_exporter] +\
                                                       holoviews.archive.exporters
-                OutputMagic.allowed['backend'] = list_backends()
-                OutputMagic.allowed['fig'] = list_formats('fig', backend)
-                OutputMagic.allowed['holomap'] = list_formats('holomap', backend)
+
+                Store.output_options = OutputOptions()
```

Comment:
Maybe although there is this:

```
    def __init__(self, *args, **kwargs):
        self.shell = kwargs.pop('shell', None)
        super(OutputOptions, self).__init__(*args, **kwargs)
        self.output.__func__.__doc__ = self._generate_docstring()
```

I can try setting ``shell`` as a class attribute and removing this, though it will lose the generated docstring.